CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 **Feature**
 * Make SP Dashboard PHP 7.2 compatible
 
+**Bugfix**
```

Comment:
I think you want to include the 2.2.3 entry in the CHANGELOG here too. Otherwise it will only be visible in the release/2.3 branch.

Reply:
Done